package-lock.json

```diff
@@ -4664,7 +4664,7 @@
 					}
 				},
 				"prettier": {
-					"version": "npm:prettier@2.2.1-beta-1",
+					"version": "npm:wp-prettier@2.2.1-beta-1",
```

Comment:
We likely need to revert this change. I believe it is erroneous. 

This particular change has been discussed a few times before. My current (untested) understanding is that `rm -rf node_modules && cd gutenberg && npm run distclean && cd ../ && npm install` may resolve the issue.

Reply:
That worked beautifully! Thanks for the tip. Fixed in 013ec06950d09a6c0176268e48dfe2b94256d33b